src/condor_tools/ping.cpp

```diff
@@ -777,6 +925,55 @@ int main( int argc, const char *argv[] )
 				usage(argv[0]);
 				exit(1);
 			}
+		} else if (is_dash_arg_prefix(argv[i],"long")) {
+			if(output_mode == -1 || output_mode == 3) {
+				output_mode = 3;
+				App.long_format = true;
+			} else {
+				fprintf(stderr,"ERROR: only one output mode may be specified.\n\n");
+				usage(argv[0]);
+				exit(1);
+			}
+		} else if (is_dash_arg_prefix (argv[i], "format", 1)) {
+			if(output_mode == -1 || output_mode == 4) {
+				output_mode = 4;
+			} else {
+				fprintf(stderr,"ERROR: only one output mode may be specified.\n\n");
+				usage(argv[0]);
+				exit(1);
+			}
+			if ( ! argv[i+1] || ! argv[i+2]) {
+				fprintf(stderr,"ERROR: -format requires 2 arguments\n\n");
+				usage(App.Name);
+				exit(1);
+			}
+			App.prmask.registerFormatF(argv[i+1], argv[i+2], FormatOptionNoTruncate);
+			if ( ! IsValidClassAdExpression(argv[i+2], &App.prattrs, NULL)) {
+				fprintf(stderr,"ERROR: %s is not a valid expression\n\n", argv[i+2]);
+				usage(App.Name);
+				exit(1);
+			}
+			output_mode = 3;
```

Comment:
Above conditional checks for `output_mode == 4` but is being set to `3`. Is the `-format` flag supposed to set output_made to `3` (long) or `4` (af)?

Reply:
line should be removed.